src/auth/authModule.ts

```diff
@@ -88,7 +88,7 @@ async function isAuthorized(jwt: any, cookie: any) {
   return !!user && !!usersClient.getUserByEmail(user.email);
 }
 
-export const auth = new Elysia()
+export const authModule = new Elysia({ name: "authModule" })
```

Comment:
Providing the name allows Elysia to dedupe duplicate installs of a module. If the module was configurable, we would also want to create a `seed` based on the input configuration so that differently configured modules don't get deduped. See [Plugin Deduplication](https://elysiajs.com/essential/plugin.html#plugin-deduplication) for more details.